index.html

```diff
@@ -6,9 +6,24 @@
     <title>Deuex</title>
     <link href="./index.css" rel="stylesheet" />
   </head>
+
   <body>
-    <nav></nav>
-    <aside></aside>
+    <aside id="sidebar">
+      <div class="sidebar-header">
+        <span class="sidebar-title">Add New Record</span>
+      </div>
+
+      <div class="buttons">
+        <button class="button" style="background-color: white">
+          <a href="#" style="text-decoration: none; color: black; padding: 2rem"
+            >Close</a
+          >
+        </button>
+        <button class="button" style="text-decoration: none">

```

Comment:
```suggestion
        <button class="button">
```
I don't think we need this style here.

Reply:
done